service/src/main/java/org/folio/rest/workflow/service/WorkflowEngineService.java

```diff
@@ -267,25 +267,29 @@ private Workflow sendWorkflowRequest(WorkflowDto workflow, String requestPath, S
         }
       }
     } catch (Exception e) {
+      log.error("Failed to send workflow request: {}!", e.getMessage(), e);
       throw new WorkflowEngineServiceException(String.format("Failed to send workflow request: %s!", e.getMessage()), e);
     }
-
+    log.error("Unable to get updated workflow from workflow engine!");
     throw new WorkflowEngineServiceException("Unable to get updated workflow from workflow engine!");
   }
 
   private HttpHeaders headers(String tenant, String token) {
     HttpHeaders requestHeaders = new HttpHeaders();
     if (tenant != null) {
+      log.info("With tenant {}", tenant);
       requestHeaders.add(tenantHeaderName, tenant);
     }
     if (token != null) {
+      log.info("With token {}", token);
```

Comment:
```suggestion
      log.debug("With token {}", token);
```

Remove this or set to `debug`.
We should not be logging a token for security reasons.

Reply:
Done.